docs/Training-ML-Agents.md

```diff
@@ -435,12 +435,11 @@ behaviors:
   # < Same as above>
 
 parameter_randomization:
-  resampling-interval: 5000
 
   mass:
-    sampler-type: "uniform"
-    min_value: 0.5
-    max_value: 10
```

Comment:
(sorry, I can't comment on the exact line)
There's a link to the now-deleted sampler_class.py below, which is causing the markdown link check to fail validation.